pom.xml

```diff
@@ -8,7 +8,7 @@
     <groupId>com.fasterxml.jackson.module</groupId>
     <artifactId>jackson-module-kotlin</artifactId>
     <name>jackson-module-kotlin</name>
-    <version>2.9.8-inin-SNAPSHOT</version>
+    <version>2.9.8.1-inin-SNAPSHOT</version>
```

Comment:
why add an increment.increment version instead of just bumping to 2.9.9?

Reply:
Because the first three versions should correlate to the Jackson release

Reply:
This will probably end up living forever... I'd prefer you just do 2.9.9 but won't block

Reply:
Why will it end up living forever? When we upgrade Jackson to 2.9.9 or 3.0.0, we can drop the 4th version and only add it back if needed.

Reply:
Nevermind, I posted that before I saw your reply to my last one